src/test/java/domain/racegame/RacingGameTest.java

```diff
@@ -0,0 +1,52 @@
+package domain.racegame;
+
+import domain.car.Car;
+import domain.car.Cars;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+class RacingGameTest {
+
+    private RacingGame racingGame;
+    Cars cars;
+
+    @BeforeEach
+    void setUp() {
+
+        List<Car> carList = Arrays.asList(
+                new Car("goldmong"),
+                new Car("ruka"),
+                new Car("pika")
+        );
+
+        cars = new Cars(carList);
+
+        racingGame = new RacingGame(cars, 3);
+    }
+
+    @Test
+    @DisplayName("첫 번째 라운드 후, 라운드가 진행 중인지 확인한다.")
+    void isRaceOngoingTestWhenRaceIsNotFinished() {
+        racingGame.playOneRoundRace();
+        Assertions.assertTrue(racingGame.isRaceOngoing());
+
+        racingGame.playOneRoundRace();
+        Assertions.assertTrue(racingGame.isRaceOngoing());
```

Comment:
[Approve]

다른 리뷰들에 비해 중요도는 떨어지지만 AssertJ에서 제공하는 검증을 사용해보는 것은 어떠신가요?
훨씬 `actual 이 expected와 어떠한지`가 잘 표현이 돼서 테스트 코드를 읽기 편해진답니다.
(org.assertj.core.api.Assertions 임폴트)

```suggestion
        final boolean actual = racingGame.isRaceOngoing();
        assertThat(actual).isEqualTo(true)
```